TimerTriggerAttachmentsVerify/index.ts

```diff
@@ -0,0 +1,27 @@
+import { AzureFunction, Context } from "@azure/functions";
+import * as df from "durable-functions";
+
+const timerTrigger: AzureFunction = async (
+  context: Context,
+  // tslint:disable-next-line: no-any
+  myTimer: any
+): Promise<void> => {
+  const timeStamp = new Date().toISOString();
+  const client = df.getClient(context);
+
+  if (myTimer.IsPastDue) {

```

Comment:
it is not clear what this part of code is meant to do, can you add some comments ?

Reply:
yes see the next PR